### PR TITLE
Fixes compilation on Power8

### LINKED ISSUE #3000: Compile Bazel on Power8
Hi!

I am trying to compile Bazel from source on a Power 8 machine.

Here's the error I get when launching "bash compile.sh"

🍃  Building Bazel from scratch.......
🍃  Building Bazel with Bazel.
ERROR: No toolchain found for cpu 'ppc'. Valid cpus are: [
  piii,
  armeabi-v7a,
  x64_windows_msvc,
  x64_windows_msys,
  s390x,
  ios_x86_64,
].
INFO: Elapsed time: 2.181s

ERROR: Could not build Bazel

The output from uname -a:

4.7.4-200.fc24.ppc64 #1 SMP Thu Sep 22 17:40:37 UTC 2016 ppc64 ppc64 ppc64 GNU/Linux

Do you have any idea of why I get this error?

Thank you

### PR DESCRIPTION
**Pull request by davideleoni90**
This fixes #3000

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: tools/cpp/cc_configure.bzl [hunk 1]
```diff
@@ -161,7 +161,7 @@ def _get_cpu_value(repository_ctx):
     return "x64_windows"
   # Use uname to figure out whether we are on x86_32 or x86_64
   result = repository_ctx.execute(["uname", "-m"])
-  if result.stdout.strip() in ["power", "ppc64le", "ppc"]:
+  if result.stdout.strip() in ["power", "ppc64le", "ppc","ppc64"]:
```
Comment (damienmg) [inline tools/cpp/cc_configure.bzl]: nit: add missing space after the comma.

**Comment by damienmg** [comment]
Hi @davideleoni90, thanks the PR looks good now except for the paperwork.

Can you sign the CLA (it takes only 2 seconds) and ping back that thread so we can import this PR?

**Comment by davideleoni90** [comment]
I have just signed the CLA: how should I proceed?

**Comment by damienmg** [comment]
Humm the first commit email is "shadyalaa <shadyalaa@gmail.com>", is it yours? In which case I believe you can also add it to the CLA signing.

**Comment by davideleoni90** [comment]
I accidentally made the commit using the github account of a collegue of mine from whom I borrow the access to the remote Power8 machine.

**Comment by damienmg** [comment]
Oh I see, so to clear things up would you mind rebasing the history? The command to do that would be: `git rebase -f HEAD~~ -x "git commit --amend --reset-author"`, and then you need to force push the . branch g`it push -f ppc64_patch`)

**Comment by damienmg** [comment]
Awesome! Thanks and sorry for all the trouble. I'll import the PR right away.

**Pull request closed without merging**